bundles/CoreBundle/Command/ClassesRebuildCommand.php

```diff
@@ -126,7 +126,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 $output->writeln(sprintf('%s saved', $brickDefinition->getKey()));
             }
 
-            $brickDefinition->save(false);
+            try {
+                $brickDefinition->save(false);
```

Comment:
fix fieldcollection as well?

Reply:
I don't think fieldcollection have this problem. There it loads all existing classes from the db.

Reply:
please see https://github.com/pimcore/pimcore/blob/c62924ca5c3d0c482a71097cb0231aff143208d6/models/DataObject/Fieldcollection/Definition/Listing.php#L27

Reply:
This loads fieldcollections and no classes

Reply:
got it....you mean the problem is with `createContainerClasses()` and since we don't have that for fieldcollection as it is other way to define it in class.

Reply:
Yes :)